src/chainstate/burn/db/sortdb.rs

```diff
@@ -756,6 +759,19 @@ impl db_keys {
         format!("sortition_db::reward_set::entry::{}", ix)
     }
 
+    pub fn pox_reward_set_payouts_key() -> String {
+        format!("sortition_db::reward_set::payouts")
+    }
+
+    pub fn pox_reward_set_payouts_value(addrs: Vec<PoxAddress>, payout_per_addr: u128) -> String {
+        serde_json::to_string(&(addrs, payout_per_addr)).unwrap()
+    }
+
+    pub fn pox_reward_set_payouts_decode(addr_str: &str) -> (Vec<PoxAddress>, u128) {
+        let addrs_and_payout: (Vec<PoxAddress>, u128) = serde_json::from_str(addr_str).unwrap();
+        addrs_and_payout
+    }
```

Comment:
Does this value need to be stored in the MARF, or can it just be indexed by the `SortitionID` in a sqlite table?

Reply:
As in, lookups are always of the form -- `what was the payout set at <sortition-id>` (which doesn't need to traverse the sortition history), and they're never `what was the payout set at <sortition-id> or if there wasn't one, what was the last payout prior?` (which could only be answered through parent traversal).

Reply:
I suppose it doesn't need to be written to the MARF.  I can just write it to a DB table -- it would make the I/O faster at least.

Reply:
Fixed